### PR TITLE
Improve robustness of find-permissions extensions

### PR DESCRIPTION
**Pull request by cd-work**
This patch adds two minor changes that should make the `find-permissions` extensions easier to use for people unfamiliar with it.

To ensure we don't incorrectly log that `/` is required when the command always fails, the extension now does an explicit check on `/` and outputs a detailed error on the issue.

While I wasn't able to reproduce this issue myself, there have been some reports on failures due to permission errors. To avoid the possibility of that we now explicitly handle `Deno.readDir` failures and just assume the corresponding directory is wholly required.

To improve consistency, all `console.error` calls have been changed to use red color.

### CODE REVIEW

File: extensions/find-permissions/main.ts [hunk 1]
```diff
@@ -107,6 +107,14 @@ for (let i = 0; true; i++) {
 // Required sandboxing exceptions.
 const requiredPaths: string[] = [];
 
+// Abort if it doesn't work with all paths allowed.
+if (!test(["/"])) {
+  let msg =
```
Comment (maxrake) [inline extensions/find-permissions/main.ts:112]:
```suggestion
  const msg =
```
This is the suggestion from the deno checks failure.

**Review by maxrake — APPROVED** [review]
I like these changes and think they will help. The only suggestion is to fix a deno checks failure.

File: extensions/find-permissions/main.ts (hunk 1 above)
Comment (cd-work) [reply extensions/find-permissions/main.ts]: Interesting, I didn't find this when running it locally. But I guess as long as CI finds...

**Review by maxrake — APPROVED** [review]
LGTM

**Pull request merged**